src/applications/vaos/sass/vaos.scss

```diff
@@ -1,6 +1,36 @@
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "../../../platform/forms/sass/m-schemaform";
 
+.vaos-appts__tabs ul.react-tabs__tab-list {
+  padding: 0;
+  border-bottom: 1px solid $color-gray-light;
+}
+
+.vaos-appts__tab {
+  width: 50%;
+  font-size: 14px;
+  border-color: $color-gray-light;
+  border: solid 1px $color-gray-light;
+  border-radius: 0 !important;
```

Comment:
Tried to use border utility classes for these, but the `!important` was messing with the selected state.

Reply:
Yeah, it's weird that we don't have tabs in the design system and need to write our styles for this.